### ISSUE TITLE
Dataset Preprocess and Alignment Details

### ISSUE THREAD

**Issue by Haoyu1004** [facebookresearch/4DGT#12]
Hi, thanks for your exciting work and opensourced demo!

I am trying to infer pretrained model on Epic-Kictchen dataset with Epic-Fields pose. However my rendered video frames is blurring, and I am wondering if it is related to preprocess steps used in your training. 

<img width="448" height="224" alt="Image" src="https://github.com/user-attachments/assets/d261525f-843f-489b-bc5d-ee1d342c0d23" />

I noticed that your model also utilzied Epic Fields in training. Would it be possible that you release demos or describe preprocess steps and hyperparameters on other training datasets used as well? Thanks.

**Comment by dendenxu** [comment]
Hi, thanks for using our code! The COLMAP poses in Epic-Fields are not metric. Although our model can handle some deviation from the true metric scale, too large a difference would still lead to very blurry results.
